icon/index.js

```diff
@@ -222,13 +227,18 @@ import {
     faAngleRight,
     faAngleUp,
     faArrowDown,
+    faArrowDownAZ,
+    faArrowDownZA,
+    faArrowDown19 as faArrowDown19,
```

Comment:
Why is the 'as' needed here?

Reply:
'As' is not required and is now removed.